setup.py

```diff
@@ -95,16 +94,14 @@ def find_meta(meta):
         package_dir={"": "src"},
         install_requires=[
             # Fix cryptographyMinimum in tox.ini when changing this!
-            "cryptography>=2.1.4",
+            "cryptography>=2.2",
             "six>=1.5.2"
         ],
         extras_require={
             "test": [
                 "flaky",
                 "pretend",
-                # pytest 3.3 doesn't support Python 2.6 anymore.
-                # Remove this pin once we drop Python 2.6 too.
-                "pytest>=3.0.1,<3.3.0",
+                "pytest",
```

Comment:
Please keep the version minimum in the pin

Reply:
OK, done!